jarbas/core/management/commands/reimbursements.py

```diff
@@ -80,16 +77,16 @@ def serialize(self, reimbursement):
 
         return reimbursement
 
-    def bulk_create_by(self, reimbursements, size):
-        batch = list()
-        for reimbursement in reimbursements:
-            batch.append(reimbursement)
-            if len(batch) == size:
-                self.bulk_create(batch)
-                batch = list()
-        self.bulk_create(batch)
-
-    def bulk_create(self, batch):
-        Reimbursement.objects.bulk_create(batch)
-        self.count += len(batch)
-        self.print_count(Reimbursement, count=self.count)
+    def create_or_update(self, reimbursements_as_dicts):
+        for count, reimbursement in enumerate(reimbursements_as_dicts):
+            document_id = reimbursement.get('document_id')
+            if document_id:
+                Reimbursement.objects.update_or_create(
+                    document_id=document_id,
+                    defaults=reimbursement
+                )
+            self.print_count(Reimbursement, count=count + 1)
```

Comment:
Wow… now I got it. Great idea! Probably there are tons of stuff to count: lines processed, actual count in the DB, how many records were included, how many were updated.

> edit: probably it's a new feature. We shouldn't hold this PR because of this.

Would you mind opening an issue with a summary so we work on that later?